advanced/events.md

```diff
@@ -142,7 +142,6 @@ The following events are provided by the {{ site.productname }} editor.
 | IconsLoadError         | `{ message: string }`                                                                                                   | Fired when the editor icon pack fails to load. {{site.requires_5_2v}}                                                                                                     |
 | LanguageLoadError      | `{ message: string }`                                                                                                   | Fired when the editor language pack fails to load. {{site.requires_5_2v}}                                                                                                  |
 | BeforeRenderUI         | N/A                                                                                                                     | Fired before the theme UI is rendered.                                                                                                                                     |
-| TextColorChange        | `{ name: string, color: string }`                                                                                       | Fired when the text color is changed.                                                                                                                                      |
```

Comment:
What are semver's thoughts on this?

Reply:
Docs don't follow semver and it's not actually removed in the source, as it's an internal-only event and as such should never have been documented. Also worth noting the description for this was wrong anyways.